### PR TITLE
fix: tr-296 fix scrollbars behavior for zoom (MSIE) in fullscreen mode

### PR DESCRIPTION
**Pull request by lecosson**
Related to:
- https://oat-sa.atlassian.net/browse/TR-296

vertical scrollbar added to page on zoom in fullscreen mode for MSIE

How to check:
- set `DEBUG_MODE` to  `false` in `config/generis.conf.php`
- check delivery in fullscreen mode with MSIE11, try to use zoom and ensure test runner controls are available

### CODE REVIEW

File: views/build/grunt/sass.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    }
+                ]
+            },
+        },
+        watch : {
+            taotestrunnerpluginssass : {
+                files : [path.join(root, 'scss/**/*.scss')],
+                tasks : ['sass:taotestrunnerplugins'],
+                options : {
+                    debounceDelay : 1000
```
Comment (btamas) [inline views/build/grunt/sass.js:48]: Why is this necessary?

File: views/js/runner/plugins/security/fullScreen.js [hunk 2]
```diff
@@ -23,7 +23,8 @@ define([
     'lodash',
     'i18n',
     'taoTests/runner/plugin',
-    'taoQtiTest/runner/config/states'
+    'taoQtiTest/runner/config/states',
+    'css!taoTestRunnerPluginsCss/fullscreen'
```
Comment (btamas) [inline views/js/runner/plugins/security/fullScreen.js:27]: Where did you register `taoTestRunnerPluginsCss` alias? How the system know what is the path?

Comment (lecosson) [reply views/js/runner/plugins/security/fullScreen.js:27]: Nowhere, it is generated by grunt task and used [same way](https://github.com/oat-sa/extension-tao-nfer/blob/fc6d1dcbae6d558f38beb1ffc0304e4af076b003/views/js/qtiCreator/plugins/panel/itemTemplates.js#L32) as in another extensions

File: views/build/grunt/sass.js (hunk 1 above)
Comment (lecosson) [reply views/build/grunt/sass.js:48]: It is not necessary, extension will works without this option, but using by grunt watch mode for stalk changes and rebuild scss on fs changes.

Comment (btamas) [reply views/build/grunt/sass.js:48]: Ok, so it was in the origin, where did you copy. I see.

File: views/js/runner/plugins/security/fullScreen.js (hunk 2 above)
Comment (btamas) [reply views/js/runner/plugins/security/fullScreen.js:27]: I'm 100% sure there is no connection with grunt task. But I do not know, where but it is automatically added to `/config/tao/client_lib_registry.conf.php`. I think Tao has an automatic solution for it.

**Review by jsconan — CHANGES_REQUESTED** [review]
The overall look good.
However, the stylesheet being related to the fullscreen plugin only, it should rather be scoped to it. The top level scss folder is used for extension wide styling. While the added one only relates to the fullScreen plugin.

**Comment by lecosson** [comment]
moved, but looks like it must be `taoTestRunnerPlugins` instead of `taoTests`

**Review by jsconan — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by meefox — APPROVED** [review]
Screenshot 2021-03-16 at 19 30 31
scroll bar is here
